client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -45,17 +63,21 @@ class RemoveDomainDialog extends Component {
 						components: { strong: <strong /> },
 					} ) }
 				</FormSectionHeading>
+
+				{ this.renderDomainDeletionWarning( productName ) }
+
 				<p>
+					{ translate( 'If you want to use this domain with another service, DO NOT delete it.' ) }{ ' ' }
```

Comment:
This language is a bit harsh. What if we say:

`If you want to use this domain with another service, you don't have to delete it. Instead, keep the domain and then move or point it to another service.`

Reply:
A string update would take at least 1 week to translate. I'm going to deploy this, but will bring it up for discussion. Thanks :)